### PR TITLE
Add support for displaying images and fixed width text.

### PR DESCRIPTION
**Pull request by msporny**

### CODE REVIEW

File: credentialMixin.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ export const credentialMixin = {
 };
 
 function _createFields(fields, source, schema) {
-  for(const key in source) {
+  for(const key in schema) {
```
Comment (msporny) [inline credentialMixin.js:105]: This ensures that fields are rendered in schema order, not object order, which was wrong.

File: CredentialCardField.vue [hunk 2]
```diff
@@ -60,7 +61,17 @@
         <span>{{name}}</span>
       </div>
       <q-item-section
-        v-if="!sublabels"
+        v-if="component === 'Image'"
+        class="g-field-data-regular">
+        <q-img style="max-width: 128px" class="rounded-borders" :src="value" />
```
Comment (mattcollier) [inline CredentialCardField.vue]: I don't know if the height should be bounded as well.

**Review by mattcollier — APPROVED** [review]
Awesome!

File: CredentialCardField.vue (hunk 2 above)
Comment (msporny) [reply CredentialCardField.vue]: Yeah, height should be bounded, fixing.

File: CredentialCardField.vue [hunk 3]
```diff
@@ -60,7 +61,17 @@
...
+      </q-item-section>
+      <q-item-section
+        v-else-if="component === 'FixedWidth'"
```
Comment (davidlehn) [inline CredentialCardField.vue:70]: If you want to haggle over naming, FixedWidthText might be better.  Or maybe PreformattedText.

**Pull request merged**